packages/grafana-ui/src/components/Flex/Flex.internal.story.tsx

```diff
@@ -0,0 +1,209 @@
+          <Item key={i} color="darkmagenta" text={i + 1} />
+        ))}
+      </Flex>
+      <p>Justify Content flex-start</p>
+      <Flex direction="row" wrap="wrap" alignItems="center" justifyContent="flex-start" gap={2}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color="darkmagenta" text={i + 1} />
+        ))}
+      </Flex>
+      <p>Justify Content flex-end</p>
+      <Flex direction="row" wrap="wrap" alignItems="center" justifyContent="flex-end" gap={2}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color="darkmagenta" text={i + 1} />
+        ))}
+      </Flex>
+      <p>Justify Content center</p>
+      <Flex direction="row" wrap="wrap" alignItems="center" justifyContent="center" gap={2}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color="darkmagenta" text={i + 1} />
+        ))}
+      </Flex>
+    </div>
+  );
+};
+
+export const GapExamples: StoryFn<typeof Flex> = () => {
+  return (
+    <div>
+      <p>Gap with spacingToken set to 2 (16px)</p>
+      <Flex direction="row" wrap="wrap" alignItems="flex-start" justifyContent="center" gap={2}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color="darkmagenta" text={i + 1} />
+        ))}
+      </Flex>
+      <p>Gap with spacingToken set to 8 (64px)</p>
+      <Flex direction="row" wrap="wrap" alignItems="flex-start" justifyContent="center" gap={8}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color="darkmagenta" text={i + 1} />
+        ))}
+      </Flex>
+      <p>Gap with spacingToken set to 10 (80px)</p>
+      <Flex direction="row" wrap="wrap" alignItems="flex-start" justifyContent="center" gap={10}>
+        {Array.from({ length: 5 }).map((_, i) => (
+          <Item key={i} color="darkmagenta" text={i + 1} />
+        ))}
+      </Flex>
+    </div>
+  );
+};
+
+export const WrapExamples: StoryFn<typeof Flex> = () => {
+  return (
+    <div>
+      <p>Wrap example with wrap and gap set to spacingToken 8 (64px)</p>
+      <Flex direction="row" wrap="wrap" alignItems="center" justifyContent="center" gap={8}>
+        {Array.from({ length: 10 }).map((_, i) => (
+          <Item key={i} color="pink" text={i + 1} />
+        ))}
+      </Flex>
+      <p>Wrap example with wrap-reverse and gap set to spacingToken 4 (32px)</p>
+      <Flex direction="row" wrap="wrap-reverse" alignItems="center" justifyContent="center" gap={4}>
+        {Array.from({ length: 15 }).map((_, i) => (
+          <Item key={i} color="pink" text={i + 1} />
+        ))}
+      </Flex>
+    </div>
+  );
+};
+
+export const DirectionExamples: StoryFn<typeof Flex> = () => {
+  return (
+    <div>
+      <p>Direction row</p>
+      <Flex direction="row" wrap="wrap" alignItems="flex-start" justifyContent="center" gap={2}>
+        {Array.from({ length: 3 }).map((_, i) => (
+          <Item key={i} color="darkmagenta" text={i + 1} />
```

Comment:
We have a problem with the colors right now in that if you switch the storybook theme between dark/light, things don't seem very accessible on light mode. I think we should use a theme color as the background here and then just white or black for the text, but to use this we actually need to stop using `Text` in the `Item`, because it doesn't allow for a text that is the same color in light and dark theme...